server/utils/auxiliaryFunctions.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
It's better to have the function declaration, and then, at the bottom, the export. Just in case you want to add more functions here. Ie:
`  checkLogBody(body) {
    return Array.isArray(body.logs) ? body.logs[0] : body;
  }

module.exports = {checkLogBody}`